build-system/tasks/e2e/describes-e2e.js

```diff
@@ -490,7 +485,6 @@ function describeEnv(factory) {
         // If there is an async expect error, throw it in the final state.
         const lastExpectError = getLastExpectError();
         if (lastExpectError) {
-          this.test.error(lastExpectError);
```

Comment:
`Mocha.Runnable.error` is not a function and unless I am missing something we never extend it so I do not believe this function actually exists. Please lmk if I am missing some thing here.

Reply:
This line was deliberately added in #22366. I don't think it should be removed until we're certain there will be no repercussions. Maybe consult with @estherkim who wrote this code, and until then, merely silence the error?

Reply:
Alright, I was a little nervous about this one (thus the comment).

Reply:
Relevant https://github.com/mochajs/mocha/issues/4515

tl;dr: it's not a documented API but works